Dockerfile

```diff
@@ -1,13 +1,26 @@
 FROM centos:7
+ARG JAVA_VERSION=1.8.0
 
-RUN yum -y install java-1.8.0-openjdk-devel && yum clean all
+RUN yum -y update \
+    && yum -y install java-${JAVA_VERSION}-openjdk-headless openssl \
+    && yum -y clean all
+
+# Set JAVA_HOME env var
 ENV JAVA_HOME /usr/lib/jvm/java
 
-ADD target/kafka-bridge.jar /
+# Add strimzi user with UID 1001
+# The user is in the group 0 to have access to the mounted volumes and storage
+RUN useradd -r -m -u 1001 -g 0 strimzi
+
+ARG strimzi_version=1.0-SNAPSHOT
+ENV STRIMZI_KAFKA_BRIDGE_VERSION ${strimzi_kafka_bridge_version}
+ENV STRIMZI_HOME=/opt/strimzi
+RUN mkdir -p /opt/strimzi/bin && mkdir -p /opt/strimzi/lib
+WORKDIR /opt/strimzi
 
-# copy scripts for starting the bridge
-COPY scripts/ /
+COPY scripts/ bin/
+COPY target/kafka-bridge.jar lib/
 
-EXPOSE 5672 5671 8080
+USER 1001
 
-CMD ["/run_bridge.sh"]
+CMD ["/opt/strimzi/bin/run_bridge.sh"]
```

Comment:
is this change coordinated with Stanislav PR, he is removing the CMD line if I recall well. He is also using a different /opt subfolder. See #206

Reply:
Well, one of us will need to reconcile. This follows the images from main repo, so I think this is much better then just pushing stuff into root.

Reply:
@stanlyDoge ^^